### DISCUSSION TITLE
Improve TLS security

### DISCUSSION THREAD

**Discussion by criena** [deta/cloud-docs#532] [Ideas]
When checking my Micro on [Qualys](https://www.ssllabs.com/ssltest/), I see a lot of issues. The most important being:

- TLS v1.0 and v1.1 are still supported (they are considered insecure nowadays).
- PFS is not fully supported.
- A lot of weak ciphers are supported.
- There seems to be something wrong with the certificate chain.

Any chance that this will improve in the near future?

**Comment by abdelhai** [comment]
hi @criena, thanks for the feedback. this is an area where we definitely want to improve. taking this to the team tomorrow. will keep this question posted.
